webpack.config.js

```diff
@@ -19,10 +19,11 @@ module.exports = {
     target: "web",
     output: {
         filename: productionBuild ? "librarie.min.js" : "librarie.js",
-        path: path.join(__dirname, '/dist/build'),
-        publicPath: "./dist/build/",
+        path: path.join(__dirname, '/dist/'),
```

Comment:
Can we preserve the build folder? This would be consistent than other web components

Reply:
Done!